assets/js/modules/analytics/common/tracking-exclusion-switches.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Analytics Tracking Exclusion switches component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../datastore/constants';
+import Switch from '../../../components/switch';
+const { useSelect, useDispatch } = Data;
+
+const TRACKING_LOGGED_IN_USERS = 'loggedinUsers';
+
+export const trackingExclusionLabels = {
+	[ TRACKING_LOGGED_IN_USERS ]: __( 'Logged-in users', 'google-site-kit' ),
+};
+
+export default function TrackingExclusionSwitches() {
+	const defaultValue = [ TRACKING_LOGGED_IN_USERS ];
+	const trackingDisabled = useSelect( ( select ) => select( STORE_NAME ).getTrackingDisabled() ) || defaultValue;
```

Comment:
This is already the default value in PHP, so it should come back as that if no value is set. I'm actually wondering why we need this default value here - if this is undefined, it should be in loading state I'd think?

Reply:
Similar to the use for `AccountSelect`, this component will raise an error below where array methods are called if the value is `undefined`. The odd thing about this one is that an empty array is a valid value which indicates user-preference to include logged in users in tracking, so the real default is what's set above.

I don't really like the idea of keeping track of the default value in multiple places like it is now, should we maybe return `null` instead if there is no value yet? Rendering a loading state is also an interesting idea @felixarntz. Ideally I think the component would render in a loading state, but I think the progress bar is a bit much for this kind of thing. It would be nice if the component rendered a loading/placeholder version of the component (think FB loading feed) if the value was `undefined`. The reality is that this state will never really be seen under normal circumstances since settings will already be loaded, and even if they weren't the REST request is preloaded so it would resolve too fast to ever see this.

I'm open to other ideas here but I think the most efficient solution here is probably to render nothing if there is no resolved value. I'll go ahead and make this change for now and we can iterate from there if needed.

Reply:
Rendering nothing was actually a thought that came to my mind too just while reading your response - and then I kept reading to see you suggested the same 😂 I agree rendering nothing for this type of component makes sense here, given the lack of a proper loading indicator.